app/browser/api/ledger.js

```diff
@@ -2419,6 +2410,30 @@ const initialize = (state, paymentsEnabled) => {
   }
 }
 
+const schedulePromoCodeRead = () => {
+  if (promoCodeReadTimeoutId) {
+    clearTimeout(promoCodeReadTimeoutId)
+  }
+
+  promoCodeReadTimeoutId = setTimeout(() => {
+    module.exports.firstRunPromoCode()
+  }, ledgerUtil.milliseconds.minute)
```

Comment:
this should be random for example 50s to 70s

Reply:
fixed